OpenMEEG/include/MEGsensors.h

```diff
@@ -0,0 +1,102 @@
+professionals having in-depth computer knowledge. Users are therefore
+encouraged to load and test the software's suitability as regards their
+requirements in conditions enabling the security of their systems and/or
+data to be ensured and,  more generally, to use and operate it in the
+same conditions as regards security.
+
+The fact that you are presently reading this means that you have had
+knowledge of the CeCILL-B license and that you accept its terms.
+*/
+
+#pragma once
+
+#include <vector>
+
+#include <IOUtils.H>
+#include <vector.h>
+#include <matrix.h>
+#include <geometry.h>
+#include <sensors.h>
+
+#include <OpenMEEG_Export.h>
+
+namespace OpenMEEG {
+
+    /*!
+     *  Sensors class for EEG and MEG sensors.
+     *  This class is made for reading sensors description file. This description file is a file text. Sensors may have names (labels)
+     *  in the first column of the file (it has to contains at least one character to be considered as label)
+     *  the file can have the shape of (neglecting if present the first, label column):
+     *  <ul>
+     *
+     *  <li> 1 line per sensor and 3 columns (MEG sensors without orientation)
+     *        <ul TYPE="circle">
+     *        <li> the 1st, 2nd and 3rd columns are respectively position coordinates x, y, z of sensor  </li>
+     *        </ul>
+     *  </li>
+     *  <li> 1 line per sensor and 6 columns (MEG sensors) :
+     *        <ul TYPE="circle">
+     *        <li> the 1st, 2nd and 3rd are respectively position coordinates x, y, z of sensor  </li>
+     *        <li> the 4th, 5th and 6th are coordinates of vector orientation </li>
+     *        </ul>
+     *  </li>
+     *  <li> 1 line per integration point for each sensor and 7 columns (MEG sensors) :
+     *        <ul TYPE="circle">
+     *        <li> the 1st, 2nd and 3rd are respectively position coordinates x, y, z of sensor  </li>
+     *        <li> the 4th, 5th and 6th are coordinates of vector orientation </li>
+     *        <li> the 7th is the weight to apply for numerical integration (uses sensor name) </li>
+     *        </ul>
+     *  </li>
+     *  </ul>
+     */
+
+    class OPENMEEG_EXPORT MEGSensors : public Sensors {
+    public:
+        MEGSensors(): Sensors() { }; /*!< Default constructor. */
+        MEGSensors(const char* filename): Sensors() { load(filename); }; /*!< Construct from file. */
+
+        void info(int n_lines = 5) const; /*!< \brief get n_lines first lines info about sensors. */
+        void load(const char* filename); /*!< Load sensors from file. */
+        void save(const char* filename);
+        Matrix& getOrientations() {return m_orientations ; } /*!< Return a reference on sensors orientations. */
+        Matrix getOrientations() const {return m_orientations ; } /*!< Return a copy of sensors orientations. */
+        bool hasOrientations() const { return m_orientations.nlin() > 0 ;} /*!< Return true if contains orientations */
```

Comment:
It cannot be False for an MEG sensor. I guess the hasOrientations method is now unecessary.